docs/development/dev-services.rst

```diff
@@ -203,9 +203,9 @@ Install dependencies
   sudo pip-2.7 install docutils
   brew install mercurial
 
-Install lessc
+Install node dependencies
 ::
-  sudo npm install -g less
+  npm install less
```

Comment:
There are two bootstrap.sh

The one in `webapp-django/bin/bootstrap.sh` will run the `npm install` in the `webapp-django` directory.